src/app/starredentries/starredentries.component.html

```diff
@@ -16,110 +16,248 @@
+            <a.no-underline routerLink="/containers/{{ tool.tool_path }}" target="_blank" rel="noopener">
+              <div fxLayout="row" fxLayoutAlign="space-between">
+                <div>
+                  <mat-card-header>
+                    <mat-card-title fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                      <img src="../../../assets/svg/sub-nav/tool.svg" alt="tool logo" />
+                      <h4>{{ tool?.namespace + '/' + tool?.name + (tool?.toolname ? '/' + tool?.toolname : '') }}</h4>
+                    </mat-card-title>
+                    <div *ngIf="tool?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center"></div>
+                  </mat-card-header>
+                </div>
+                <app-starring class="pull-right" [tool]="tool" (starGazersChange)="starGazersChange()"></app-starring>
+              </div>
+              <div>
+                <hr />
+                <div fxLayout="row" fxLayoutAlign="space-between stretch">
+                  <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                    <small class="spacing">Last updated {{ tool?.lastUpdated | date }}</small>
+
+                    <mat-chip-list>
+                      <mat-chip class="container-label-tag" *ngFor="let label of tool?.labels | slice: 0:3">{{ label.value }}</mat-chip>
+                      <mat-chip class="gray-label-tag" *ngFor="let descriptortype of tool?.descriptorType">
+                        {{ descriptortype }}
+                      </mat-chip>
+                    </mat-chip-list>
 
-        <mat-list>
-          <mat-list-item *ngFor="let workflow of starredWorkflows; let last = last">
-            <h3 class="spacing" mat-line>
-              <a routerLink="/workflows/{{ workflow.full_workflow_path }}" target="_blank" rel="noopener">
-                {{ workflow?.organization + '/' + workflow?.repository + (workflow?.toolname ? '/' + workflow?.toolname : '') }}
-              </a>
-            </h3>
-            <span mat-line>
-              <small class="spacing">Last updated {{ workflow?.lastUpdated | date }}</small>
-              <span class="spacing">{{ workflow?.descriptorType | uppercase }}</span>
-              <a class="spacing" *ngIf="workflow?.provider" [href]="workflow?.providerUrl">{{ workflow?.provider }}</a>
-            </span>
-            <app-starring
-              class="pull-right"
-              style="width: 100px"
-              [workflow]="workflow"
-              (starGazersChange)="starGazersChange()"
-            ></app-starring>
-            <mat-divider></mat-divider>
-          </mat-list-item>
-        </mat-list>
+                    <span id="verifiedIcon" *ngIf="(tool$ | async)?.versionVerified" fxHide.lt-sm>
+                      <mat-chip matTooltip="Verified" class="gray-label-tag">
+                        <a [href]="verifiedLink" class="verified-check" fxLayoutGap="0.5rem" fxLayoutAlign=" center">
```

Comment:
I was going to suggest this should be a routerLink instead of an href. But then I went into the editor to see if it was an internal vs. external link (it should be routerLink only if it's internal), and the IDE says both `verifiedLink` and `tool$` on line 152 are undefined. IDE could be funky, but it seems like something is wrong -- can you please check?

Reply:
Yeah, I mentioned on the old ticket that I was having some trouble getting the verified versions for the workflow and tools, so I must've forgotten to change that. I made a ticket for that, but don't think it's resolved yet. I also just removed the href, since I don't think it is needed. Do you think I should just remove the verified check entirely for now?